### DISCUSSION TITLE
How do I sign commits using the VS Code source control UI?

### DISCUSSION THREAD

**Discussion by batmanscode** [community/community#27558] [Codespaces] [answered]
I enabled GPG verification following this: https://docs.github.com/en/codespaces/managing-your-codespaces/managing-gpg-verification-for-github-codespaces

**Comment by jfullstackdev** [comment, marked as answer]
I'm not using VS Code, but if you can create a local branch and publish it to GitHub original repo, then, you will be able to verify your commits without any additional verification, because the one that will be merging it on GitHub is you, right there 

this is true for GitHub Desktop

**Reply by batmanscode** [reply]
Ah I see. So the commits to the branch are not verified but the merge gets verified? Seems like a decent work around. Thank you 😃
